### PR TITLE
Create BaseSection and Highlight sections

### PR DESCRIPTION
**Pull request by grosbouddha**
- Create BaseSection and HighlightSection sections
- Improve naming for section sizes in the branding file

image

TESTED=Locally in storybook.

### CODE REVIEW

File: src/layout/section/highlightSection/story.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import HighlightSection from './index'
+import BlankSeparator from 'blankSeparator'
+
+const stories = storiesOf('Sections|HighlightSection', module)
+stories.addDecorator(withKnobs)
+
+stories.add('default', () => (
+  <Fragment>
+    <BlankSeparator />
+    <HighlightSection>This a highlighted section !</HighlightSection>
```
Comment (Tuxosaurus) [inline src/layout/section/highlightSection/story.tsx]: definitely not nit: no whitespace before " ! "

File: src/layout/section/baseSection/baseSection.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  readonly role?: string
+  readonly children: JSX.Element | string
+  readonly contentSize?: SectionContentSize
+}
+
+/**
+ * The core section: It sections horizontally a page while fitting its
+ * content in width-constrained and centered column.
+ */
+class BaseSection extends PureComponent<BaseSectionProps> {
```
Comment (wakooka) [inline src/layout/section/baseSection/baseSection.tsx]: Functional component instead

File: src/layout/section/highlightSection/highlightSection.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+export interface HighlightSectionProps {
+  readonly className?: Classcat.Class
+  readonly children: JSX.Element | string
+}
+
+/**
+ * A specialized section with an highlighting background color.
+ */
+class HighlightSection extends PureComponent<HighlightSectionProps> {
```
Comment (wakooka) [inline src/layout/section/highlightSection/highlightSection.tsx]: Functional component instead

File: src/layout/section/highlightSection/story.tsx (hunk 1 above)
Comment (grosbouddha) [reply src/layout/section/highlightSection/story.tsx]: Done !

File: src/layout/section/baseSection/baseSection.tsx (hunk 2 above)
Comment (grosbouddha) [reply src/layout/section/baseSection/baseSection.tsx]: Done

File: src/layout/section/highlightSection/highlightSection.tsx (hunk 3 above)
Comment (grosbouddha) [reply src/layout/section/highlightSection/highlightSection.tsx]: Done

**Pull request merged**